nms/fbc_js_core/ui/host/Organizations.js

```diff
@@ -231,13 +146,28 @@ function Organizations(props: Props) {
     [classes.index],
   );
 
-  const renderLinkColumn = useCallback(rowData => {
-    return (
-      <Link href={rowData.portalLink}>
-        {`Visit ${rowData.name} Organization Portal`}
-      </Link>
-    );
-  }, []);
+  const renderLinkColumn = useCallback(
+    rowData => {
+      return (
+        <Link
+          href={rowData.portalLink}
+          variant="body2"
+          ref={
+            organizations?.length === rowData.tableData?.index + 1 &&
+            organizations?.length < 3 &&
+            //hide onboarding helper for host organization
+            !(rowData.name === 'host')
```

Comment:
Does that mean the helper will not be shown when last organization happens to be the host.
We could either always put the `host` on top if we want to make sure the popup is shown below the table. Or we just position the popup on the `magma-test` organization.